examples/cpp-debug-workspace/README.md

```diff
@@ -6,6 +6,8 @@
 
 The workspace provides an easy and reproducible way to test the functionality present in the `@theia/cpp-debug` extension.
 
+note: ATM debugging does not work in Gitpod. Since GDB uses ptrace to attach to the debuggee and ptrace requires the docker container to be started with special permissions.
```

Comment:
[minor] typo on `debugee` (debugger)

Reply:
Actually not a typo :)  

https://en.wiktionary.org/wiki/debuggee

Reply:
> Actually not a typo :)
> 
> https://en.wiktionary.org/wiki/debuggee

Haha I was wondering if you mean the debuggee or the debugger itself 🙈